prometheusbp/promtest/testutil.go

```diff
@@ -1,4 +1,4 @@
-package prometheusbp
+package promtest
```

Comment:
github does not allow me to add comments further down this file as they are not changed in this PR, but I think we should add "Use MetricTest to construct it" in the doc comment of `PrometheusMetricTest`, or rename `MetricTest` into `NewPrometheusMetricTest`.